packages/components/psammead-media-player/README.md

```diff
@@ -32,6 +35,19 @@ The `portrait` prop is not required, and defaults to `false`. This is to support
 The `showPlaceholder` boolean prop is also not required, and defaults to `true`.
 Assuming `showPlaceholder` is `true`, the `placeholderSrc` will be what image to display as the placeholder.
 
+#### mediaInfo
```

Comment:
This is great. Can I suggest we link to it from the above `mediaInfo` line?
- The link should be to `#mediainfo`.